src/resqml2_0_1/RockFluidUnitFeature.h

```diff
@@ -50,7 +50,18 @@ namespace RESQML2_0_1_NS
 		*/
 		~RockFluidUnitFeature() {}
 
+		DLL_IMPORT_OR_EXPORT void setFluidBoundaryTop(class FluidBoundaryFeature* fluidBoundaryFeature);
```

Comment:
Ideally, it should not constrain to use FluidBoundaryFeature. It would be better to use its BoundaryFeature super class.